README.rst

```diff
@@ -9,30 +9,195 @@ modelmeta
+
+  - In particular, Alembic can be used to *create* a new instance of a ``modelmeta`` database by migrating an
+    empty database to the current state. This is described in detail below.
+
+For more information, see the `Alembic tutorial <http://alembic.zzzcomputing.com/en/latest/tutorial.html>`_.
+
+History
+-------
+
+The existing instance of a ``modelmeta`` database (``monsoon/pcic_meta``) was created prior to the adoption of
+Alembic, and therefore the timeline for Alembic database migrations is slightly confusing.
+
+Timeline:
+
+- *the distant past*: ``pcic_meta`` is created by mysterious primeval processes.
+
+- *somewhat later*: ``modelmeta`` is defined using SQLAlchemy, mapping most (but not all) features of the existing
+  ``pcic_meta`` database into an ORM.
+
+- 2017-07-18:
+
+    - Alembic is introduced.
+    - Alembic is used to create migration ``614911daf883`` that adds item ``seasonal`` to ``timescale`` Enum.
+
+- 2017-08-01:
+
+    - The SQLAlchemy ORM is updated to reflect all features of the ``pcic_meta`` database.
+      This mainly involves adding some missing indexes and constraints.
+
+    - Alembic is used to create a logically-previous migration ``7847aa3c1b39`` that creates the initial
+      database schema from an empty database.
+
+    - The add-seasonal migration is modified to logically follow the initial-create migration.
+
+Creating a new ``modelmeta`` database
+-------------------------------------
+
+There are in fact two tools in this package to create a new ``modelmeta`` database.
+
+- (DEPRECATED) Use the script ``scripts/mkblankdb.py``.
+  The resulting database will not be under Alembic migration management.
+- (RECOMMENDED) Use Alembic, as described below.
+
+Creating a new database with Alembic
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For a Postgres database
++++++++++++++++++++++++
+
+A Postgres database is somewhat more elaborate to set up, but it is also the foundation of a production
+database, not least because we use PostGIS.
+
+These instructions follow the pattern of database name and schema name established in ``pcic_meta``:
+
+- The name of the database and of the schema which houses the ``modelmeta`` tables proper are the same,
+  e.g., ``pcic_meta``.
+
+This is not by any means mandatory.
+
+Instructions:
+
+#. Choose a name for your new database/schema, e.g., ``ce_meta``.
+
+#. On the server of your choice (e.g., ``monsoon``):
+
+    a. Create a new user, with either of the following properties:
+
+        - the username is the same as the schema, e.g., ``ce_meta``,
+          with the default search path (``"$user",public``), OR
+        - it has a default search path of the form ``<name>,public``, e.g., ``ce_meta,public``
+
+    #. Create a new database with the chosen name, e.g., ``ce_meta``.
+
+    #. Within that database, create a new schema with the chosen name, e.g., ``ce_meta``.
+
+    #. Somehow install PostGIS in there. How??
```

Comment:
Pretty much comes down to issuing ``CREATE EXTENSION POSTGIS`` at this point, but you could probably link to the PostGIS install docs.